src/main/java/net/commments/object/CommentWindow.java

```diff
@@ -4,11 +4,21 @@
 
     void fillCommentTextField(String commentText);
 
+    void fillCommentNumber(String commentNumber);
+
     void saveComment();
 
+    void saveCommentReturnInTable();
+
     String showErrorMessage();
 
     void addCategory();
 
     boolean isErrorMessageShown(String errorMessage);
```

Comment:
The method has to return a current value of the error message field. The comparison has to happen within the test.

Reply:
fixed